google_bigquery_writer/app.py

```diff
@@ -100,7 +87,8 @@ def get_writer(self):
 
         bigquery_client_factory = BigqueryClientFactory(
             project,
-            self.get_credentials()
+            self.get_credentials(),
+            location=self.cfg.get_parameters().get('location', None)
```

Comment:
Nemělo toto být náhodou součástí toho service_account objektu?

Možná by stálo za to rovnou upravit i ten if else blok nad tím na:

```
config_data_service_account = self.cfg.config_data.get('image_parameters', {}).get('service_account', {})
parameters_service_account = self.cfg.get_parameters().get('service_account')

project = (
    config_data_service_account.get('project_id') or
    (parameters_service_account and parameters_service_account.get('project_id')) or
    self.cfg.get_parameters().get('project')
)
```

To je asi přehlednější.

Reply:
- pod jedním service accountem může být více datasetů s různou lokalitou
- if else zjednodušeno, teď díky validaci parametrů i pro image parametry mnohem přehlednější